pegasus/sites.v3/code.org/public/congrats.haml

```diff
@@ -30,8 +30,9 @@ max_age: 60
   end
 
   facebook = {:u=>share_url}
-  twitter = {:url=>share_url, :related=>'codeorg', :text=>I18n.t('just_did_hoc_donor', donor_twitter: get_random_donor_twitter)}
-  twitter[:hashtags] = 'HourOfCode' unless I18n.t(:just_did_hoc_donor).include? '#HourOfCode'
+  twitter_message = I18n.t('just_did_hoc_donor', donor_twitter: get_random_donor_twitter)
```

Comment:
How does this fix the interpolation error?

Reply:
As explained offline, the fix is the 2nd call to the I18n API which doesn't pass `donor_twitter`.